src/components/Header.tsx

```diff
@@ -47,24 +49,46 @@ const NavWrapper = styled.nav`
   }
 `;
 
-const Header = () => {
+function Header() {
+  const navigate = useNavigate();
+  const { isAuthenticated } = useAuth();
+
+  const handleSignOut = async () => {
+    await signOut();
+    navigate('/');
+  };
+
   return (
     <HeaderWrapper>
       <Link to="/">
         <h1>Phisical Exercise</h1>
```

Comment:
I suspect you intend this to be "Physical Exercise" rather than "Phisical Exercise"?